Arkweb-Proposal-Maxium-Fling-Speed.md

```diff
@@ -0,0 +1,41 @@
+# Maxium fling speed
+
+## Authors:
+
+- Qin Wen Jun (Arkweb)
+
+
+## Introduction
+
+This proposal aims to establish a declarative mechanism `HTMLMetaElement.maxFlingSpeed ` to limit fling speed on websites with complex dom structure.   
+
+## User-Facing Problem
+
+Modern websites now feature increasingly complex DOM structures with rich functionalities. Many sites host a vast amount of media resources, such as images and videos, while adopting infinite-scrolling designs on mobile for a seamless experience. However, when a page is overloaded, rapid user scrolling (e.g., "flicking" gestures) can strain the rendering engine, potentially causing blank screens due to delayed rendering. Moreover, excessively fast scrolling negatively impacts user experience.
+
+### Goals
+
+- This proposal introduces two new attributes, max-fling-speed-x and max-fling-speed-y, to the content field of the <meta name="viewport"> tag. These attributes accept ​​positive integer values​​ (in pixels per second, px/s) to cap the maximum fling speed in horizontal (X-axis) and vertical (Y-axis) scrolling, respectively.
+
```

Comment:
In case the author sets eg `max-fling-speed-x=1`, I think the UA should clamp it to some reasonable value. The exact value can be UA-defined.